### PR TITLE
Framework: disable starred sites while we keep testing recent sites.

### PR DESCRIPTION
**Pull request by mtias**
Let's test recent sites in isolation for a little longer as it just hit desktop app.

### CODE REVIEW

**Comment by aduth** [comment]
I suppose we don't care to preserve starring for those who have already opted in to the feature?

**Comment by mtias** [comment]
Not for now in the UI, but we'll keep the data.

File: client/lib/sites-list/list.js [hunk 1]
```diff
@@ -567,10 +569,10 @@ SitesList.prototype.getVisible = function() {
 SitesList.prototype.getVisibleAndNotRecentNorStarred = function() {
 	return this.get().filter( function( site ) {
 		if ( user.get().visible_site_count < 12 ) {
-			return site.visible === true && ! this.isStarred( site );
```
Comment (aduth) [inline client/lib/sites-list/list.js]: Yeah, it's a quick patch, but still bothers me that the naming of the function is now inconsistent with the return value of the function. Would have personally rather have seen this function left as is, with another nearly identical for `getVisibleAndNotRecent` .

File: client/my-sites/site/index.jsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ module.exports = React.createClass( {
 
 			homeLink: false,
 			enableActions: false,
-			disableStarring: false
+			disableStarring: true
```
Comment (aduth) [inline client/my-sites/site/index.jsx]: `enableStarring: false` would be nice for avoiding the "not not" syndrome.

**Comment by aduth** [comment]
Confirmed sites list reflects recentness, ignores starring, that editing toggle is still available but not offered option to star site, and that star status is not indicated in the UI. :+1:

File: client/lib/sites-list/list.js (hunk 1 above)
Comment (mtias) [inline client/lib/sites-list/list.js]: I could do this for our sanity :)

**Pull request merged**